libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -960,7 +962,7 @@ void RsGxsNetService::handleRecvSyncGrpStatistics(RsNxsSyncGrpStatsItem *grs)
 
 	    RsNxsSyncGrpStatsItem *grs_resp = new RsNxsSyncGrpStatsItem(mServType) ;
 	    grs_resp->request_type = RsNxsSyncGrpStatsItem::GROUP_INFO_TYPE_RESPONSE ;
-	    grs_resp->number_of_posts = vec.size();
+	    grs_resp->number_of_posts = static_cast<uint32_t>(vec.size());
```

Comment:
is that really necessary?

Reply:
You're right, number_of_posts should be size_t. ;)
/libretroshare/src/gxs/rsgxsnetservice.cc:963: warning: implicit
conversion loses integer precision: 'std::vector::size_type' (aka
'unsigned long') to 'uint32_t' (aka 'unsigned int')
size_t depends on system.